5.0/fr/0x12-V3-Web-Frontend-Security.md

```diff
@@ -0,0 +1,100 @@
+| **3.5.5** | Vérifiez que les messages reçus par l'interface postMessage sont rejetés si l'origine du message n'est pas fiable ou si la syntaxe du message n'est pas valide. | 2 |
+| **3.5.6** | Vérifiez que la fonctionnalité JSONP n'est activée nulle part dans l'application pour éviter les attaques Cross-Site Script Inclusion (XSSI). | 3 |
+| **3.5.7** | Vérifiez que les données nécessitant une autorisation ne sont pas incluses dans les réponses des ressources de script, comme les fichiers JavaScript, pour empêcher les attaques par inclusion de script intersite (XSSI). | 3 |
+| **3.5.8** | Vérifiez que les ressources authentifiées (telles que les images, les vidéos, les scripts et autres documents) peuvent être chargées ou intégrées pour le compte de l'utilisateur uniquement lorsque cela est prévu. Cela peut être réalisé en validant strictement les champs d'en-tête de requête HTTP Sec-Fetch-* afin de garantir que la requête ne provient pas d'un appel cross-origin inapproprié, ou en définissant un champ d'en-tête de réponse HTTP Cross-Origin-Resource-Policy restrictif pour indiquer au navigateur de bloquer le contenu renvoyé. | 3 |
+
+## V3.6 Intégrité des ressources externes
+
+Cette section fournit des conseils pour l’hébergement sécurisé de contenu sur des sites tiers.
+
+| # | Description | Niveau |
+| :---: | :--- | :---: |
+| **3.6.1** | Vérifiez que les ressources côté client, telles que les bibliothèques JavaScript, les feuilles de style CSS ou les polices web, sont hébergées en externe (par exemple, sur un réseau de diffusion de contenu) uniquement si la ressource est statique et versionnée, et que l'intégrité des sous-ressources (SRI) est utilisée pour valider l'intégrité de la ressource. Si cela n'est pas possible, une décision de sécurité documentée doit justifier cette décision pour chaque ressource. | 3 |
+
+## V3.7 Autres considérations sur la sécurité du navigateur
+
+Cette section comprend divers autres contrôles de sécurité et fonctionnalités de sécurité de navigateur modernes nécessaires pour la sécurité du navigateur côté client.
+
+| # | Description | Niveau |
+| :---: | :--- | :---: |
+| **3.7.1** | Vérifiez que l'application utilise uniquement des technologies côté client toujours prises en charge et considérées comme sécurisées. Parmi les technologies qui ne répondent pas à cette exigence figurent les plugins NSAPI, Flash, Shockwave, ActiveX, Silverlight, NACL ou les applets Java côté client. | 2 |
+| **3.7.2** | Vérifiez que l'application redirigera automatiquement l'utilisateur vers un nom d'hôte ou un domaine différent (qui n'est pas contrôlé par l'application) lorsque la destination apparaît sur une liste blanche. | 2 |
```

Comment:
There is something missing here to make the meaning clear:

"Vérifiez que l'application redirigera automatiquement l'utilisateur vers un nom d'hôte ou un domaine différent (qui n'est pas contrôlé par l'application) *uniquement* lorsque la destination apparaît sur une liste blanche."